### DISCUSSION TITLE
No module named 'selenium'

### DISCUSSION THREAD

**Discussion by ssspetrovic** [TobiasPankner/Teams-Auto-Joiner#160] [Q&A] [answered]
Hello, when I run the final python command to open the auto joiner i get this error: C:\Users\spetr\AppData\Local\Programs\Python\Python310\Teams-Auto-Joiner-master>python auto_joiner.py
Traceback (most recent call last):
  File "C:\Users\spetr\AppData\Local\Programs\Python\Python310\Teams-Auto-Joiner-master\auto_joiner.py", line 8, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'

Even though I installed the prerequisites. I get this when i try to install selenium separately:
Requirement already satisfied: selenium in c:\users\spetr\appdata\local\programs\python\python310\lib\site-packages (3.141.0)
Requirement already satisfied: urllib3 in c:\users\spetr\appdata\local\programs\python\python310\lib\site-packages (from selenium) (1.26.7)

**Comment by TobiasPankner** [comment, marked as answer]
Hi.
Please try to run it with python3, or try installing the dependencies with pip3 install ...

**Reply by ssspetrovic** [reply]
Still no result, i already have Python 3.10 installed.

C:\Users\spetr\Desktop\Teams-Auto-Joiner-master>pip3 install selenium
Requirement already satisfied: selenium in c:\users\spetr\appdata\local\programs\python\python310\lib\site-packages (3.141.0)
Requirement already satisfied: urllib3 in c:\users\spetr\appdata\local\programs\python\python310\lib\site-packages (from selenium) (1.26.7) 

C:\Users\spetr\Desktop\Teams-Auto-Joiner-master>python3 auto_joiner.py
Traceback (most recent call last):
  File "C:\Users\spetr\Desktop\Teams-Auto-Joiner-master\auto_joiner.py", line 8, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'

**Reply by TobiasPankner** [reply]
Please run `python3 -v`

**Reply by ssspetrovic** [reply]
Seems like it's using the one that came with MINGW that i had installed in the past, 
import _frozen_importlib # frozen
import _imp # builtin
import '_thread' # <class '_frozen_importlib.BuiltinImporter'>
import '_warnings' # <class '_frozen_importlib.BuiltinImporter'>
import '_weakref' # <class '_frozen_importlib.BuiltinImporter'>
import '_io' # <class '_frozen_importlib.BuiltinImporter'>
import 'marshal' # <class '_frozen_importlib.BuiltinImporter'>
import 'nt' # <class '_frozen_importlib.BuiltinImporter'>
import 'winreg' # <class '_frozen_importlib.BuiltinImporter'>
import '_frozen_importlib_external' # <class '_frozen_importlib.FrozenImporter'>
# installing zipimport hook
import 'time' # <class '_frozen_importlib.BuiltinImporter'>
import 'zipimport' # <class '_frozen_importlib.FrozenImporter'>
# installed zipimport hook
# C:\msys64\mingw64\lib\python3.9\encodings\__pycache__\__init__.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\encodings\__init__.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\encodings\\__pycache__\\__init__.cpython-39.pyc'
# C:\msys64\mingw64\lib\python3.9\__pycache__\codecs.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\codecs.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\__pycache__\\codecs.cpython-39.pyc'
import '_codecs' # <class '_frozen_importlib.BuiltinImporter'>
import 'codecs' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f034820>
# C:\msys64\mingw64\lib\python3.9\encodings\__pycache__\aliases.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\encodings\aliases.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\encodings\\__pycache__\\aliases.cpython-39.pyc'
import 'encodings.aliases' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f1a0d90>
import 'encodings' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f034550>
# C:\msys64\mingw64\lib\python3.9\encodings\__pycache__\utf_8.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\encodings\utf_8.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\encodings\\__pycache__\\utf_8.cpython-39.pyc'
import 'encodings.utf_8' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f034a60>
# C:\msys64\mingw64\lib\python3.9\encodings\__pycache__\cp1252.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\encodings\cp1252.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\encodings\\__pycache__\\cp1252.cpython-39.pyc'
import 'encodings.cp1252' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f1a0e80>
import '_signal' # <class '_frozen_importlib.BuiltinImporter'>
# C:\msys64\mingw64\lib\python3.9\encodings\__pycache__\latin_1.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\encodings\latin_1.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\encodings\\__pycache__\\latin_1.cpython-39.pyc'
import 'encodings.latin_1' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f1ba220>
# C:\msys64\mingw64\lib\python3.9\__pycache__\io.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\io.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\__pycache__\\io.cpython-39.pyc'
# C:\msys64\mingw64\lib\python3.9\__pycache__\abc.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\abc.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\__pycache__\\abc.cpython-39.pyc'
import '_abc' # <class '_frozen_importlib.BuiltinImporter'>
import 'abc' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f1ba6d0>
import 'io' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f1ba400>
# C:\msys64\mingw64\lib\python3.9\__pycache__\site.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\site.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\__pycache__\\site.cpython-39.pyc'
# C:\msys64\mingw64\lib\python3.9\__pycache__\os.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\os.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\__pycache__\\os.cpython-39.pyc'
# C:\msys64\mingw64\lib\python3.9\__pycache__\stat.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\stat.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\__pycache__\\stat.cpython-39.pyc'
import '_stat' # <class '_frozen_importlib.BuiltinImporter'>
import 'stat' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f1d9fa0>
# C:\msys64\mingw64\lib\python3.9\__pycache__\_collections_abc.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\_collections_abc.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\__pycache__\\_collections_abc.cpython-39.pyc'
import '_collections_abc' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f1d9e20>
# C:\msys64\mingw64\lib\python3.9\__pycache__\ntpath.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\ntpath.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\__pycache__\\ntpath.cpython-39.pyc'
# C:\msys64\mingw64\lib\python3.9\__pycache__\genericpath.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\genericpath.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\__pycache__\\genericpath.cpython-39.pyc'
import 'genericpath' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f20a8e0>
import 'ntpath' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f1e19a0>
import 'os' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f1cd6d0>
# C:\msys64\mingw64\lib\python3.9\__pycache__\_sitebuiltins.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\_sitebuiltins.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\__pycache__\\_sitebuiltins.cpython-39.pyc'
import '_sitebuiltins' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f1d93a0>
# C:\msys64\mingw64\lib\python3.9\__pycache__\sysconfig.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\sysconfig.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\__pycache__\\sysconfig.cpython-39.pyc'
# C:\msys64\mingw64\lib\python3.9\__pycache__\textwrap.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\textwrap.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\__pycache__\\textwrap.cpython-39.pyc'
# C:\msys64\mingw64\lib\python3.9\__pycache__\re.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\re.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\__pycache__\\re.cpython-39.pyc'
# C:\msys64\mingw64\lib\python3.9\__pycache__\enum.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\enum.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\__pycache__\\enum.cpython-39.pyc'
# C:\msys64\mingw64\lib\python3.9\__pycache__\types.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\types.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\__pycache__\\types.cpython-39.pyc'
import 'types' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f23da00>
import 'enum' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f230550>
# C:\msys64\mingw64\lib\python3.9\__pycache__\sre_compile.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\sre_compile.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\__pycache__\\sre_compile.cpython-39.pyc'
import '_sre' # <class '_frozen_importlib.BuiltinImporter'>
# C:\msys64\mingw64\lib\python3.9\__pycache__\sre_parse.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\sre_parse.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\__pycache__\\sre_parse.cpython-39.pyc'
# C:\msys64\mingw64\lib\python3.9\__pycache__\sre_constants.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\sre_constants.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\__pycache__\\sre_constants.cpython-39.pyc'
import 'sre_constants' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f35f7c0>
import 'sre_parse' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f24bdf0>
import 'sre_compile' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f24b550>
# C:\msys64\mingw64\lib\python3.9\__pycache__\functools.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\functools.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\__pycache__\\functools.cpython-39.pyc'
# C:\msys64\mingw64\lib\python3.9\collections\__pycache__\__init__.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\collections\__init__.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\collections\\__pycache__\\__init__.cpython-39.pyc'
# C:\msys64\mingw64\lib\python3.9\__pycache__\heapq.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\heapq.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\__pycache__\\heapq.cpython-39.pyc'
# extension module '_heapq' loaded from 'C:\\msys64\\mingw64\\lib\\python3.9\\lib-dynload\\_heapq.cp39-mingw_x86_64.pyd'
# extension module '_heapq' executed from 'C:\\msys64\\mingw64\\lib\\python3.9\\lib-dynload\\_heapq.cp39-mingw_x86_64.pyd'
import '_heapq' # <_frozen_importlib_external.ExtensionFileLoader object at 0x000001ba6f3a2550>
import 'heapq' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f39bf70>
import 'itertools' # <class '_frozen_importlib.BuiltinImporter'>
# C:\msys64\mingw64\lib\python3.9\__pycache__\keyword.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\keyword.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\__pycache__\\keyword.cpython-39.pyc'
import 'keyword' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f3a2640>
# C:\msys64\mingw64\lib\python3.9\__pycache__\operator.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\operator.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\__pycache__\\operator.cpython-39.pyc'
import '_operator' # <class '_frozen_importlib.BuiltinImporter'>
import 'operator' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f3a26d0>
# C:\msys64\mingw64\lib\python3.9\__pycache__\reprlib.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\reprlib.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\__pycache__\\reprlib.cpython-39.pyc'
import 'reprlib' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f3a27c0>
import '_collections' # <class '_frozen_importlib.BuiltinImporter'>
import 'collections' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f383430>
import '_functools' # <class '_frozen_importlib.BuiltinImporter'>
import 'functools' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f36ccd0>
import '_locale' # <class '_frozen_importlib.BuiltinImporter'>
# C:\msys64\mingw64\lib\python3.9\__pycache__\copyreg.cpython-39.pyc matches C:\msys64\mingw64\lib\python3.9\copyreg.py
# code object from 'C:\\msys64\\mingw64\\lib\\python3.9\\__pycache__\\copyreg.cpython-39.pyc'
import 'copyreg' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f3836d0>
import 're' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f223ac0>
import 'textwrap' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f2234c0>
import 'sysconfig' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f1d9940>
import 'site' # <_frozen_importlib_external.SourceFileLoader object at 0x000001ba6f1baf70>
Python 3.9.7 (default, Sep 23 2021, 09:13:41)  [GCC 10.3.0 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
import 'atexit' # <class '_frozen_importlib.BuiltinImporter'>

**Reply by TobiasPankner** [reply]
Yes, you need to change your env variables to use the new version.

**Reply by ssspetrovic** [reply]
Thank you very much.
